File: Manuel/2025-08-05/OX퀴즈/8958.cpp
#include <iostream>
#include <string>
#include <vector>
#define endl "\n"

using namespace std;

int main()
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);

    int T;
    cin >> T; // 테스트 케이스의 수 입력

    for(int i = 0; i < T; i++)
    {
        string result;
        cin >> result; // OX 퀴즈 결과 문자열 입력

        int total_score = 0;
        int streak = 0;

        // 문자열을 순회하며 점수 계산
        for(char c : result)
        {
            if(c == 'O' || c == 'o')
            {
                streak++;
                total_score += streak;
            }
            else
            {
                streak = 0;
            }
        }
        cout << total_score << endl;
    }
    return 0;
}

File: Manuel/2025-08-05/OX퀴즈/8958.py
import sys

def main():
    input = sys.stdin.readline

    T = int(input())
    for _ in range(T):
        result = input().strip()
        total_score = 0
        streak = 0

        for char in result:
            if char == 'O' or char == 'o':
                streak += 1
                total_score += streak
            else:
                streak = 0
        print(total_score)

main()